cmd/openshift-install/gather.go

```diff
@@ -50,7 +50,19 @@ func newGatherBootstrapCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "bootstrap",
 		Short: "Gather debugging data for a failing-to-bootstrap control plane",
-		Args:  cobra.ExactArgs(0),
+		Long: `Gather debugging data for a failing-to-bootstrap control plane.
+
+If the Terraform state file is present, the installer will attempt to
```

Comment:
Do not include Terraform into any docs or user-facing doc text. How installer fetches ips for IPI based installs is not user-facing... like users should not care we use terraform under the hood.

Reply:
> Do not include Terraform...

Dropped with 31c55e7fa -> 197e13b2b.